tailwind.config.js

```diff
@@ -2,7 +2,14 @@
 module.exports = {
   content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      height: {
+        '29': '29rem',
+      },
+      fontSize: {
+        'xxs': '0.5rem',
```

Comment:
no this doesn't looks goods keep it `0.5: 0.5em` is better , please make this change in a followup PR